pinot-common/src/main/java/org/apache/pinot/common/metadata/segment/LLCRealtimeSegmentZKMetadata.java

```diff
@@ -87,11 +87,6 @@ public void setDownloadUrl(String downloadUrl) {
   public ZNRecord toZNRecord() {
     ZNRecord znRecord = super.toZNRecord();
     znRecord.setSimpleField(START_OFFSET, _startOffset);
-    if (_endOffset == null) {
```

Comment:
Please flag the commit for release notes and add in checkin comments that in order for people to use this release, all components should have been upgraded to the previous release (or, 0.5.0, not sure. please verify). thanks